src/utils/constants.js

```diff
@@ -10,3 +10,6 @@ export const DAILY_FORECAST_TITLE_TEXT = 'Daily forecast symbol for';
 
 export const CURRENT_CITY_FORECAST_ALT_TEXT = 'Current city forecast';
 export const CURRENT_CITY_FORECAST_TITLE_TEXT = 'Current city forecast';
+
+export const EMPTY_WARNING_LIST_MAIN_MESSAGE = 'Warnings list is empty';
+export const EMPTY_WARNING_LIST_ADDITIONAL_MESSAGE = 'Have a nice day!';
```

Comment:
This is overhead. Either you use i18n and create json vocabularies. Or just use strings as is. They are one of a kind and everybody understands that that is a user text